extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -1277,13 +1307,12 @@ angular.module('oppia').component('svgFilenameEditor', {
             var y = ctrl.polyOptions.y;
             ctrl.polyOptions.bboxPoints.push(new polyPoint(x, y));
             var points = [x, y, x, y];
-            var size = ctrl.fabricjsOptions.size;
             var stroke = ctrl.fabricjsOptions.stroke;
             // This is to ensure that the polygon lines are visible when
             // creating the polygon.
             stroke = stroke.slice(0, -2) + '1)';
             var line = new fabric.Line(points, {
-              strokeWidth: parseInt(size.substring(0, size.length - 2)),
+              strokeWidth: getStrokeWidth(),
               selectable: false,
               stroke: stroke,
               strokeLineCap: 'round'
```

Comment:
Can you please add detailed comments for each of the `ctrl.canvas.on(..)`s defined below explaining what they do?

Reply:
Done.